docs/analyzer/user_guide.md

```diff
@@ -99,7 +99,8 @@ usage: CodeChecker check [-h] [-o OUTPUT_DIR] [-t {plist}] [-q]
                          [--report-hash {context-free,context-free-v2}]
                          [-i SKIPFILE | --file FILE [FILE ...]]
                          [--analyzers ANALYZER [ANALYZER ...]]
-                         [--capture-analysis-output] [--config CONFIG_FILE]
+                         [--capture-analysis-output] [--generate-reproducer]
```

Comment:
Please update the help output of the CodeChecker analyze command too.

Reply:
Done